src/component/Map/Map.tsx

```diff
@@ -127,6 +133,20 @@ export class Map extends React.Component<MapProps, MapState> {
     }
   }
 
+  /**
+   * Called on lifecycle componentWillUnmount.
+   */
+  componentWillUnmount() {
+    const map = this.props.map;
+    map.un('moveend', this.onMapMoveEnd);
+    map.un('change:view', this.onMapViewChange);
+    document.removeEventListener('mousemove', (evt) => {
+      this.setState({
+        mouseEvt: evt
+      });
+    });
```

Comment:
Isn't this a different function than when it was registered? Should we pass the function as a reference, e.g. from a variable or such?

Reply:
fixed